html2text/__init__.py

```diff
@@ -905,6 +905,8 @@ def optwrap(self, text):
                     indent = ""
                     if para.startswith("  " + self.ul_item_mark):
                         indent = "    "  # For list items.
+                    elif para.startswith("> "):
```

Comment:
If you could put some comments here and describe the reasoning as comment, it would be really really good.

Reply:
Is that ok ?

Reply:
Yes, it does!